assemble/bin/accumulo-cluster

```diff
@@ -129,21 +147,9 @@ function control_service() {
     [[ "$service" == "compactor" ]] && ACCUMULO_SERVICE_INSTANCE="${inst_id}_${5}"
 
     if [[ $host == localhost || $host == "$(hostname -s)" || $host == "$(hostname -f)" || $host == $(get_ip) ]] ; then
-      #
-      # The server processes take arguments (e.g. -a, -p, -o, -q [in the case of the Compactor]).
-      #
-      if [[ $# -gt 3 ]]; then
-        ACCUMULO_SERVICE_INSTANCE="${ACCUMULO_SERVICE_INSTANCE}" "${bin}/accumulo-service" "$service" "$control_cmd" "${@:4}"
-      else
-        ACCUMULO_SERVICE_INSTANCE="${ACCUMULO_SERVICE_INSTANCE}" "${bin}/accumulo-service" "$service" "$control_cmd"
-      fi
+      ACCUMULO_SERVICE_INSTANCE="${ACCUMULO_SERVICE_INSTANCE}" "${bin}/accumulo-service" "$service" "$control_cmd" "$EXTRA_ARGS"
```

Comment:
This won't work as intended. `EXTRA_ARGS` is scalar here, so the quoting will prevent multiple extra args from being recognized as separate arguments. It previously would have worked, because `@` is an array, so quoting that is equivalent to it being replaced by each of its elements being individually quoted. This is a bash quirk regarding arrays.

If you want this to work the way it did before, you have to make `EXTRA_ARGS` an array, and quote it like `"${EXTRA_ARGS[@]}"`. But, it is used as a scalar in other places, so you'd need to change those to be `"${EXTRA_ARGS[*]}"`. I think keeping it simple and avoiding the array syntax was a goal, and why it was written the way it was before.